packit/sync.py

```diff
@@ -106,6 +106,10 @@ def sync_files(files_to_sync: List[RawSyncFilesItem]) -> None:
         if src.exists():
             if src.is_dir():
                 logger.debug("src is a dir, using copytree")
+                if dest.is_dir():
+                    logger.debug("dest dir exists, rmtree it first")
+                    shutil.rmtree(dest, ignore_errors=True)
+                logger.info(f"Copying tree {src} to {dest}.")
```

Comment:
I'd put this log line before the if condition so it's clear before the rmtree what dir is being removed

Reply:
It's not visible from this snippet, but few lines above is
```python
logger.debug(f"src = {src}, dest = {dest}")
```
so you know what `dest` is, but I can add it to my debug log as well.
I won't move this one since I want it to be here (right before we call `shutil.rmtree`)